MetaTools/generate_glyphdata.py

```diff
@@ -242,11 +112,14 @@ def generate_python_source(data, out):
         out.write('\t"%s": %s,\n' % (glyphName, glyphsCat))
     out.write("}\n\n")
 
+    out.write(
+        "DEFAULT_GLYPH_DICT = %s\n\n" % glyphs
```

Comment:
not that file size matters to me -- but the current `glyphdata_generated.py` is now 8.7 MB, whereas the sum of the original `GlyphData.xml` and `GlyphData_Ideographs.xml` files is 5.9 MB.